test/support/factories.js

```diff
@@ -7,18 +7,29 @@ factory.define('userNotAdmin', User, {
   lastName: factory.chance('string')(),
   email: 'email@wolox.com.ar',
   password: 'password',
-  isAdmin: false
+  isAdmin: false,
+  invalidateTime: 0
 });
 
 factory.define('userAdmin', User, {
   firstName: factory.chance('string')(),
   lastName: factory.chance('string')(),
   email: 'emailAdmin@wolox.com.ar',
   password: 'password',
-  isAdmin: true
+  isAdmin: true,
+  invalidateTime: 0
 });
 
 factory.define('purchase', Purchase, {
   userId: 1,
   albumId: 1
 });
+
+factory.define('user', User, {
```

Comment:
Whats the point of this factory when you already have the other two?

Reply:
Also, take a look at the [extend](https://github.com/aexmachina/factory-girl#extending-factories) function from factory girl. Using it would significantly reduce the boilerplate here.